### PR TITLE
Setup project configuration

### PR DESCRIPTION
**Pull request by Millionhat**
Added .github folder with codeowners file, also included package.json

### CODE REVIEW

**Comment by garzuzo** [comment]
the commit message should be "setup project configuration" in lowercase

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "type": "git",
+    "url": "git+https://github.com/Millionhat/cypress-training.git"
+  },
+  "keywords": [
+    "ui-testing",
+    "dojo",
+    "practice",
+    "cypress"
+  ],
+  "author": "Juan Sebastian Palma <juanspalma@gmail.com> ([@Millionhat])",
```
Comment (garzuzo) [inline package.json]: remove brackets around your github user

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+@saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
```
Comment (garzuzo) [inline .github/CODEOWNERS]:
```suggestion
* @saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
```

Comment (garzuzo) [reply .github/CODEOWNERS]: **Remember:** Each line is a file pattern followed by one or more owners. **The asterisk means every file.**

Comment (Millionhat) [reply .github/CODEOWNERS]: Added new commit with edited code owners file

File: package.json (hunk 1 above)
Comment (Millionhat) [reply package.json]: removed brackets in the new revised commit

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1 +1 @@
-@saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
\ No newline at end of file
+* @saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
```
Comment (holgiosalos) [inline .github/CODEOWNERS]: EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon:
image

Comment (Millionhat) [reply .github/CODEOWNERS]: new Revised commit added the line at the end of file in code owners file

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**